app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
 <h1>Specific user</h1>
+<div class='user-class'>
+  <span class='title'> <%= @user.name %> </span> <span class='number-of-posts'>Number of posts: <%= @user.post_counter %></span>
+</div>
```

Comment:
- [ ] Same here, You have a typo here making an error, It's supposed to be posts_counter instead of post_counter, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/223888280-b67d6b15-77c9-47c6-9f57-add7571ac56b.png)